### PR TITLE
Update of submit-a-pr.md for DCO

### PR DESCRIPTION
**Pull request by spotz**
Updated instructions with workaround for DCO failure. New PR for https://github.com/o3de/o3de.org/pull/821 which was closed.

Signed-off-by: Amy Marrich (spotz) <amy@demarco.com>

### CODE REVIEW

**Comment by spotz** [comment]
@willihay @micronAMZN - the older PR was closed so created a new one. Tagging you both as I wasn't able to add reviewers.

File: content/docs/contributing/to-docs/submit-a-pr.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Despite only having five steps, the PR process can take some time depending on t
...
 
+{{< important >}}
+If you used the web UI to make your commit vs the commandline, your PR may fail the DCO check and you will need to add a `Signed-off-by:` line to your commit message. For GitHub accounts made prior to July 18, 2017, you need to use `username@users.noreply.github.com` as the email address. For accounts created after that date, you will need to use your GitHub-provided no-reply email address is a seven-digit ID number and your username in the form of `ID+username@users.noreply.github.com`. For more information regarding setting your commit email address, refer to the [Setting your commit email address instructions](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address).
```
Comment (micronAMZN) [inline content/docs/contributing/to-docs/submit-a-pr.md:46]:
```suggestion
If you use the GitHub web UI to make your commit, you must add a `Signed-off-by:` line to your commit message or the DCO check will fail. For GitHub accounts created prior to July 18, 2017, use `username@users.noreply.github.com` as the email address. For accounts created after that date, use your GitHub-provided no-reply email address. The new no-reply email address is a seven-digit ID number and your username in the form of `ID+username@users.noreply.github.com` that can be found in the email tab of your GitHub account settings. For more information about setting your commit email address, refer to the [Setting your commit email address instructions](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address).
```

**Review by micronAMZN — CHANGES_REQUESTED** [review]

**Review by spotz — COMMENTED** [review]
Thanks!

**Comment by micronAMZN** [comment]
> 
> 
> Thanks!

This commit failed the DCO :) Could you amend the last commit with a sign-off?

**Comment by spotz** [comment]
This got stuck ironically in a DCO loop so closing. Will work on getting documentation so folks don't get in a loop:)

**Pull request closed without merging**